js_basics/task5-1.js

```diff
@@ -0,0 +1,4 @@
+const radius = 25;
+const x = 2
```

Comment:
what is x? it's not meaningful name. Either change variable name or use literal 2 directly in Math.pow without defining a variable